<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QDialog" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(104, 161, 229);</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>30</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(104, 161, 229);</string>
   </property>
   <property name="text">
    <string>My health account </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(104, 161, 229);</string>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>391</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { 
     border-radius: 5px; 
	background-color: rgb(242, 248, 253);
 } 


</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditFirstName">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditLastName">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditEmail">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
      <property name="text">
       <string>NHI Number</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditNHINumber">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;background-color: rgb(242, 248, 253,0)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
      <property name="text">
       <string>Password(Confirm)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditPasswordConfirm">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 248, 253)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>163</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { 
     border-radius: 5px; 
	background-color: rgb(242, 248, 253);
 } 


</string>
   </property>
   <property name="title">
    <string>Password</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>33</y>
      <width>133</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 248, 253,0)</string>
    </property>
    <property name="text">
     <string>- At least 8 characters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>57</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 248, 253,0)</string>
    </property>
    <property name="text">
     <string>- Uppercase (A-Z)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>81</y>
      <width>107</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 248, 253,0)
</string>
    </property>
    <property name="text">
     <string>- Lowercase (a-z)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>105</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 248, 253,0)</string>
    </property>
    <property name="text">
     <string>- Number (0-9)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>129</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 248, 253,0)</string>
    </property>
    <property name="text">
     <string>- Symbols (#,$,%,&amp;,!)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirm">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
